src/notation/view/widgets/editstyle.ui

```diff
@@ -6041,7 +6041,7 @@
                   <item row="0" column="2">
                    <widget class="QCheckBox" name="singleMMRestShowNumber">
                     <property name="text">
-                     <string>Show number ‘1’</string>
+                     <string>Show number '1'</string>
```

Comment:
For another alternative solution, see `editstyle.ui` line 865.
Basically, in the .ui file, use boring quotes and noTr; then in the cpp file set the translatable version of the string, with curly quotes.

Reply:
https://github.com/musescore/MuseScore/blob/020391e58d8d9004a6769e4f882ff567a932a5fd/src/notation/view/widgets/editstyle.ui#L865 ??

Reply:
Ah, I see, https://github.com/musescore/MuseScore/blob/020391e58d8d9004a6769e4f882ff567a932a5fd/src/notation/view/widgets/editstyle.cpp#L865
So done deal, just needs the ` notr="true"`, I'll add that

Reply:
Ah, no, that's another string, so more work to be done...

Reply:
done